build.psm1

```diff
@@ -2114,6 +2116,27 @@ function Start-CrossGen {
     }
 }
 
+# Cleans the PowerShell repo
+# by default everything but the root folder and the Packages folder
+# if you specify -IncludePackages it will clean the Packages folder
+function Start-PSCleanRepo
```

Comment:
Resolved

---
In reply to: [77045786](https://github.com/PowerShell/PowerShell/pull/2139#discussion_r77045786) [](ancestors = 77045786)